### PR TITLE
Add tags display for Inspector

### PR DESCRIPTION
**Pull request by deltakosh**
Fixing https://forum.babylonjs.com/t/display-tags-in-the-inspector/42781/3

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/commonPropertyGridComponent.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import type { LockObject } from "shared-ui-components/tabs/propertyGrids/lockObj
 import type { GlobalState } from "../../../globalState";
 import { TextLineComponent } from "shared-ui-components/lines/textLineComponent";
 import { IndentedTextLineComponent } from "shared-ui-components/lines/indentedTextLineComponent";
+import { Tags } from "core/Misc";
```
Comment (Popov72) [inline packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/commonPropertyGridComponent.tsx:11]:
```suggestion
import { Tags } from "core/Misc/tags";
```

**Review by Popov72 — DISMISSED** [review]

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15822/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15822/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15822/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15822/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15822/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15822/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15822/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15822/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15822/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15822/merge/testResults/webgpuplaywright/index.html

**Review by sebavan — APPROVED** [review]

**Comment by HoferMarkus** [comment]
@deltakosh thanks a lot for the implementation!

However it seems like the tags UI is now only available for meshes and transform nodes.
AFAIK tags are available for any kind of object in Babylon.js, so couldn't we use a more generic approach like it's done for metadata?
image

If this is not possible, could we add tags for materials?
Meshes, transform nodes and materials are the important ones for us at Combeenation 😄

**Comment by deltakosh** [comment]
I'm out today but I will move it to all objects this Monday

**Pull request merged**